Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h

```diff
@@ -3424,12 +3425,56 @@ bool AbstractInterpreter<AbstractStateType>::executeEffects(unsigned clobberLimi
         makeHeapTopForNode(node);
         break;
 
-    case TryGetById:
+    case TryGetById: {
+        // This is very adhoc, but @tryGetById is not usually used in user code, and it is used adhocly in very limited places.
```

Comment:
Seems like you don't need the "usually"?

Reply:
Right, dropped.